WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+
+What is a WEP?
+--------------
+
+WEP stands for Write the Docs Enhancement Proposal. A WEP is a document that provides process information to the Write the Docs community or that describes a new policy for the Write the Docs organization.
+
+WEPs describe **policies and processes** surrounding Write the Docs events and online community, and therefore require community consensus. For example, a WEP might propose a change to an organizational process, community policies, or the decision-making process itself. 
+
+Scope of WEPs
+-------------
+
+WEPs are generally intended for long-term policies that affect either a large number of community members, or that affect fewer but more significant segments of the community.
+
+Examples of topics that should use the WEP process:
+
+- Adjusting the structure of the Slack community to make it more accessible to new community members
+- Policies about the structure of the Write the Docs organization
+- Policy related to Slack team or channel guidelines
+- Changes to the Code of Conduct (CoC)
+- A grant policy to support conference attendees that need financial support
+- A policy on how to balance various topics submitted to the Call for Proposals at the various Write the Docs conferences
+
+Examples of topics that are out of the scope of the WEP process:
+- Requests to change CFP policies in a specific conference
+- Requests to create a new Slack channel
+- Specific CoC violation reports
+- Requests to create a new meetup
+
+Topics that are out of scope for a WEP should be discussed in other community platforms, such as within the conference team, Slack team, or Code of Conduct team. The WEP team will direct out-of-scope pre-proposals to the relevant team in the rejection comments.
+
+Generally, WEPs do not contain the names of individual people, but rather references to teams in the Write the Docs organization. Similarly, they don’t contain fixed budget numbers, but instead guidelines for how to implement the budget. 
+
+How to submit a WEP
+-------------------
+
+Anyone from the community can submit a WEP! Here's how it works, and what to expect.
+
+The WEP process and system are administered by the following teams:
+
+Core Operations team
+  There are several reference in this WEP to the **core operations (CoreOps) team**. This team is currently known as the **core team** and the team description can be found on the `Core team page <https://www.writethedocs.org/team/#core-team>`_. The updated name is intended to reflect the purpose of the team more accurately in the context of its duties for the community. 
+
+WEP team
+  The **WEP team** is a new team that reviews WEP proposals for clarity and completeness. For the purpose of this WEP the team is initially composed of the Core Operations team, but in the the future will be open for other community members to join. 
```

Comment:
I do not know what this means:

> For the purpose of this WEP the team is initially composed of the Core Operations team, but in the the future will be open for other community members to join.

Does this mean that the WEP team will change after WEP 1, or will it require a WEP to include others on the WEP team?

Reply:
Or until there's a WEP to establish the membership process.. ?

Reply:
Yea, the goal is to have teams defined in an upcoming WEP after this. It's a bit circular, but couldn't think of a better way to do it.

Reply:
The future WEP should also include definitions around joining and leaving teams, but I don't htink we need to get into such a level of detail in the scope of this WEP.